### ISSUE TITLE
Accessibility Support

### ISSUE THREAD

**Issue by foxpcteam** [vitmalina/w2ui#2310]
Hi, i see w2ui have lack of accessibility support.
And this day's it's very important to support accessibility wherever possible.
i think the refactoring of V2.0 should take care of accessibility to.
